src/sabre_swap/swap_scores.rs

```diff
@@ -0,0 +1,62 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2022
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use pyo3::prelude::*;
+
+use indexmap::IndexMap;
+
+/// An unsigned integer Vector based layout class
+///
+/// This class tracks the layout (or mapping between virtual qubits in the the
+/// circuit and physical qubits on the physical device) efficiently
+///
+/// Args:
+///     qubit_indices (dict): A dictionary mapping the virtual qubit index in the circuit to the
+///         physical qubit index on the coupling graph.
+///     logical_qubits (int): The number of logical qubits in the layout
+///     physical_qubits (int): The number of physical qubits in the layout
+#[pyclass(module = "qiskit._accelerate.sabre_swap")]
+#[pyo3(text_signature = "(qubit_indices, logical_qubits, physical_qubits, /)")]
```

Comment:
Looks like a copy-paste that needs updating :).

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7977/commits/ce0ca5111f9af353c683fd4371f78ab386d8675e